README.md

```diff
@@ -114,6 +114,41 @@ JSONSchemer.schema(
 )
 ```
 
+## Parsing errors
+To get a list of errors map the results like this.
+``` 
+require 'pp'
+
+schemer = JSONSchemer.schema(your-json-schema)
+
+result = schemer.validate({
+	Values you're inputting
+})
+
+# use pp to make it nice to look at in your console.
+pp result.map { |error| error.slice('data', 'data_pointer', 'type') }
+
+```
+
+## Local file references
+
+To use local file refereces in your schema you will need to provide an absolute path to JSONSchemer and a function to resolve those references.
+
+``` 
+# refspec is the reference provided by your schema. 
+# from {"$ref": "./secondFile.json"}, -> ./secondFile.json
+
+ref_resolver = lambda do |refspec|
```

Comment:
I think you should be able to rely on the default ref resolver if you pass in a `Pathname`: https://github.com/davishmcclurg/json_schemer/blob/ab5ea07e21633952f5025aaabcfbb37c05c3a74d/lib/json_schemer.rb#L42-L48